nix/hosts/webforge/web-landing-page.nix

```diff
@@ -28,43 +28,42 @@
     enable = true;
 
     # Configure the virtualhosts to serve the pages
-    # TODO: Replace 87b59b92.nip.io by tahoe-lafs.org below when ready - trac#4162
     virtualHosts = {
       # Define a live site to serve the content generated from the main branch
       # See https://github.com/tahoe-lafs/web-landing-page
-      "live.www.87b59b92.nip.io" = {
+      "home.of.tahoe-lafs.org" = {
```

Comment:
@hacklschorsch, you've previously mentioned `www.of.tahoe-lafs.org` [here](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/4162#comment:23): we can stick to that if you prefer.